src/main/kotlin/eu/europa/ec/eudi/openid4vci/internal/formats/CredentialIssuerMetadataJsonParser.kt

```diff
@@ -547,5 +547,6 @@ private fun cryptographicBindingMethodOf(s: String): CryptographicBindingMethod
         s == "jwk" -> CryptographicBindingMethod.JWK
         s == "cose_key" -> CryptographicBindingMethod.COSE
         s.startsWith("did") -> CryptographicBindingMethod.DID(s)
+        s == "x5c" -> CryptographicBindingMethod.X5C
```

Comment:
Please remove it. Not in the specification

Reply:
done